datalad/__init__.py

```diff
@@ -112,4 +112,8 @@ def teardown_package():
     for path in _TEMP_PATHS_GENERATED:
         rmtemp(path, ignore_errors=True)
 
+    lgr.debug("Printing versioning information collected so far")
+    from datalad.support.external_versions import external_versions as ev
+    print(ev.dumps(query=True))
```

Comment:
I am a little puzzled why this one is not covered since it is in global teardown and  was ran:  https://travis-ci.org/datalad/datalad/jobs/192419096#L2699